source/_docs/configuration/yaml.markdown

```diff
@@ -1,13 +1,17 @@
 ---
-title: "YAML"
-description: "Details about YAML to configure Home Assistant."
+title: "YAML syntax"
+description: "Details about the YAML syntax used to configure Home Assistant."
 ---
 
-Home Assistant uses the [YAML](https://yaml.org/) syntax for configuration. YAML might take a while to get used to but is powerful in allowing you to express complex configurations.
+Home Assistant uses the [YAML](https://yaml.org/) syntax for configuration. While most integrations can be configured through the UI, some integrations require you to edit your [`configuration.yaml`](/docs/configuration/) file to specify its settings.
 
-While more and more integrations are configured through the UI, for some, you will add code in your [`configuration.yaml`](/docs/configuration/) file to specify its settings.
+## YAML Style Guide
 
-The following example entry assumes that you would like to set up the [notify integration](/integrations/notify) with the [pushbullet platform](/integrations/pushbullet).
+This page gives a high-level introduction to the YAML syntax used in Home Assistant. For a more detailed description and more examples, refer to the [YAML Style Guide for Home Assistant developers](https://developers.home-assistant.io/docs/documenting/yaml-style-guide/).
```

Comment:
Do we want to send end users to our developer website? In the past we kept this strictly separate.

Reply:
Excellent question. Yes, I agree with that in principle. And I was very hesitant putting that link there... But...

I think the Style Guide is a not actually a technical dev documentation, describing developpery things like implementing an async component. It is a Style Guide like the MS Style Guide, but for YAML. The topics discussed in the YAML Style Guide are topics you are going to be exposed to when you start messing with YAML in HA. The reason we don't want to send people to the dev docs is because the topics there are usually dev topics. But if we have users mess with YAML, this Style Guide gives more context and examples than what we currently have in the user docs. I think not providing that information is not a good option.

The alternative would be creating such a Guide in the User Docs. But I think the result would be a copy of the existing YAML Guide. This is probably a better solution from a user perspective. But it comes with the maintenance cost of duplicate (or similar but different) content.